programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1663,8 +1664,7 @@ static struct option run_host_options[] = {
         { "refresh", optional_argument, NULL, 'r'},
 };
 
-static const char* ip_range = "100.64.0.0/10";
-static const char* dns_impl = NULL;
+static const char* default_cidr = "100.64.0.1/10";
```

Comment:
rename "ip_range" to "default_cidr" and change the default to 100.64.0.1 which will become the default IP of the TUN if none are supplied on the command line or via config file